Frameworks/UIKit/UIApplication.mm

```diff
@@ -1258,301 +992,6 @@ - (void)_sendHighMemoryWarning {
-
-    [_blankView setFrame:startRect];
-    [popupRotationLayer addSubview:_blankView];
-
-    [UIView beginAnimations:@"ResizeAnimation" context:nil];
-
-    if (newHeight <= 0) {
-        //  Keyboard is being hidden
-        [UIView setAnimationDelegate:self];
-        [UIView setAnimationDidStopSelector:@selector(_keyboardDismissed)];
-    }
-    [_blankView setFrame:endRect];
-    [UIView commitAnimations];
-
-    [popupWindow bringSubviewToFront:popupRotationLayer];
-
-    CGRect keyboardBounds;
-    keyboardBounds.origin.x = 0.0f;
-    keyboardBounds.origin.y = 0.0f;
-    keyboardBounds.size = endRect.size;
-
-    CGPoint centerBegin, centerEnd;
-
-    centerBegin.x = mappedStart.origin.x + mappedStart.size.width / 2.0f;
-    centerBegin.y = mappedStart.origin.y + mappedStart.size.height / 2.0f;
-
-    centerEnd.x = mappedEnd.origin.x + mappedEnd.size.width / 2.0f;
-    centerEnd.y = mappedEnd.origin.y + mappedEnd.size.height / 2.0f;
-
-    // Fire the notification:
-    id keys[7] =
-    { @"UIKeyboardFrameEndUserInfoKey",
-      @"UIKeyboardFrameBeginUserInfoKey",
-      @"UIKeyboardAnimationDurationUserInfoKey",
-      @"UIKeyboardAnimationCurveUserInfoKey",
-      @"UIKeyboardBoundsUserInfoKey",
-      @"UIKeyboardCenterBeginUserInfoKey",
-      @"UIKeyboardCenterEndUserInfoKey",
-    };
-    id values[7] = {
-        [NSValue valueWithCGRect:mappedEnd],  [NSValue valueWithCGRect:mappedStart],    [NSNumber numberWithDouble:0.25],
-        [NSNumber numberWithInt:0],           [NSValue valueWithCGRect:keyboardBounds], [NSValue valueWithCGPoint:centerBegin],
-        [NSValue valueWithCGPoint:centerEnd],
-    };
-
-    id dict = [NSDictionary dictionaryWithObjects:values forKeys:keys count:7];
-
-    static float oldHeight = 0;
-
-    if (newHeight > 0) {
-        if (!blankViewUp || forceKeyboardAppearance) {
-            [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillShowNotification" object:nil userInfo:dict];
-            [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardDidShowNotification" object:nil userInfo:dict];
-        }
-        if (!blankViewUp) {
-        } else {
-            if (oldHeight != newHeight) {
-                [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillChangeFrameNotification"
-                                                                    object:nil
-                                                                  userInfo:dict];
-                [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardDidChangeFrameNotification"
-                                                                    object:nil
-                                                                  userInfo:dict];
-            }
-        }
-    } else {
-        //  Keyboard is being hidden
-        [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillHideNotification" object:nil userInfo:dict];
```

Comment:
Since applications can be written to re-layout due to keyboard presence, we should log an issue to propagate those notifications properly. Right now we're just removing them.

Alternatively, are we just dropping support for UIKit keyboard resize?